DR42 Release Notes (#438)
NCI-GDC/gdc-docs: docs/Data/Release_Notes/Data_Release_Notes.md

| [v2.0](Data_Release_Notes.md#data-release-20) | August 9, 2016 |
| [v1.0](Data_Release_Notes.md#initial-data-release-10) | June 6, 2016 |

## Data Release 42.0

* __GDC Product__: Data
* __Release Date__: January 2#, 2025

### New Updates

* New Data Sets

>     * Release of new WGS variant calling workflows. See the [documentation](Data/Bioinformatics_Pipelines/DNA_Seq_WGS.md) on WGS variant calling for more details on the available files. This includes data from the following projects:

Reviewer: ```suggestion
    * Release of new WGS variant calling workflows for existing WGS tumor normal pairs. See the [documentation](Data/Bioinformatics_Pipelines/DNA_Seq_WGS.md) on WGS variant calling for more details on the available files. This includes data from the following pipelines:
```

>         * APOLLO-LUAD

Reviewer: ```suggestion
        * GATK4 MuTect2 - SNVs (raw and annotated VCFs)
        * SvABA Indel - SNVs (raw and annotated VCFs)
        * SvABA - Structural variants (VCF and BEDPE)
        * Manta - Structural variants (VCF and BEDPE)
        * GATK4 CNV - Copy number segments and auxiliary files
        * __Note:__ Variants from pipelines were released from aliquots pairs on a pipeline-basis.  Aliquot pairs may have data from some pipelines but not others.  The completed set will be made available in future releases.
```

        * TCGA-GBM
        * TCGA-HNSC
        * TCGA-KICH
        * TCGA-KIRC
        * TCGA-KIRP
        * TCGA-LGG
        * TCGA-LIHC
        * TCGA-LUAD
        * TCGA-LUSC
        * TCGA-MESO
        * TCGA-OV
        * TCGA-PAAD
        * TCGA-PCPG
        * TCGA-PRAD
        * TCGA-READ
        * TCGA-SARC
        * TCGA-SKCM
        * TCGA-STAD
        * TCGA-TGCT
        * TCGA-THCA
        * TCGA-THYM
        * TCGA-UCEC
        * TCGA-UCS
        * TCGA-UVM
        * WCDT-MCRPC
    * TARGET-AML RNA-Seq - 46 aliquots
    * TARGET-NBL WGS - 146 raw CGI variants released
    * TCGA-GBM RNA-Seq - 215 aliquots
    * TCGA-GBM miRNA-Seq - 265 aliquots
    * TCGA-LUSC miRNA-Seq - 10 aliqiuots
    * TCGA-LUSC RNA-Seq - 9 aliquots
    * TCGA-OV miRNA-Seq - 76 aliquots
    * TCGA WGS alignments
        * TCGA-BLCA - 9 alignments
        * TCGA-BRCA - 1 alignment
        * TCGA-COAD - 54 alignments
        * TCGA-HNSC - 5 alignments
        * TCGA-KIRC - 837 alignments
        * TCGA-KIRP - 6 alignments
        * TCGA-LGG - 2 alignments
        * TCGA-LUAD - 175 alignments
        * TCGA-LUSC - 12 alignments
        * TCGA-OV - 8 alignments
        * TCGA-SKCM - 624 alignments

>     * New clinical data for all TCGA projects

Reviewer: ```suggestion
    * _New clinical data for all TCGA projects:_ Fields that were previously only available in clinical supplement files are now queryable and downloadable through the GDC Data Portal and API.  
```

| [v2.0](Data_Release_Notes.md#data-release-20) | August 9, 2016 |
| [v1.0](Data_Release_Notes.md#initial-data-release-10) | June 6, 2016 |

## Data Release 42.0

* __GDC Product__: Data
* __Release Date__: January 2#, 2025

### New Updates

* New Data Sets
    * Release of new WGS variant calling workflows for existing WGS tumor normal pairs. See the [documentation](Data/Bioinformatics_Pipelines/DNA_Seq_WGS.md) on WGS variant calling for more details on the available files. This includes data from the following pipelines:

>         * GATK4 MuTect2 - SNVs (raw and annotated VCFs)

Reviewer: It didn't seem to register this as a bullet point - at least in terms of syntax coloring.  It's probably okay, let's just make sure to check this when it renders.

    * TCGA-GBM miRNA-Seq - 265 aliquots
    * TCGA-LUSC miRNA-Seq - 10 aliqiuots
    * TCGA-OV miRNA-Seq - 76 aliquots
    * TARGET-AML RNA-Seq - 46 aliquots
    * TCGA-GBM RNA-Seq - 215 aliquots
    * TCGA-LUSC RNA-Seq - 9 aliquots
    * TARGET-NBL WGS - 146 raw CGI variants released
    * New versions of 31 CTSP-DLBCL1 clinical supplements

* Data Updates
    * Tumor purity and tumor ploidy properties were migrated from the [aligned reads node](https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=aligned_reads) to the [copy number segment node](https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=copy_number_segment) <!--DAT-3683-->

A complete list of files included in the GDC Data Portal can be found below:

* [gdc_manifest_2025012#_data_release_42.0_active.tsv.gz](https://api.gdc.cancer.gov/data/<uuid>)
* [DR42 Project Level Manifests](https://api.gdc.cancer.gov/data/<uuid>)
* [DR42 New Files Manifest](https://api.gdc.cancer.gov/data/<uuid>)

### Bugs Fixed Since Last Release

* Pathology reports now have associated case/biospecimen information in the portal.

> 

Reviewer: ```suggestion
* Several bugs were fixed with the inclusion of TCGA clinical data in the API.

```

* The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.
* Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.
* Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.
* BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.
* No data from TARGET-MDLS is available.
* TCGA Projects

>     * Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265-->

Reviewer: ```suggestion

```

    * 74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111-->
    * Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439-->

>     * The raw and annotated VarScan VCF files for aliquot `TCGA-VR-A8ET-01A-11D-A403-09` are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489-->

Reviewer: ```suggestion
   
```

    * Some TCGA annotations are unavailable in the Data Portal<!--DAT-52-->. These annotations can be found [here](tcga-annotations-unavailable-20170315.json).

>     * Tumor_grade property is not populated <!--SV-585-->

Reviewer: ```suggestion
   
```

>     * Progression_or_recurrence property is not populated <!--SV-584-->

Reviewer: ```suggestion
  
```
